docs/withdrawals/FAQs.md

```diff
@@ -0,0 +1,26 @@
+---
+stoplight-id: 9ji36bqx57o2j
+---
+
+# Withdrawal FAQs
+
+## What should be used as a withdrawal's reference?
+
+The only limitation is a maximum of 50 characters, otherwise this is up to you. You might choose to generate something unique or partially unique concatenated with something more readable for your users to be able to tell withdrawals apart.
```

Comment:
Does this reference get put on the withdrawal? If so will it be truncated if longer than a certain number of characters?

Reply:
Good point... how about something like this? technically a lie as we know it's our bank that is doing the truncating but leaves the docs open to when we move payments to ClearBank.
```suggestion
The only limitation is a maximum of 50 characters, otherwise this is up to you. You might choose to generate something unique or partially unique concatenated with something more readable for your users to be able to tell withdrawals apart. Note that this reference could be truncated by the customers bank.
```

Reply:
We could just say that it could get truncated by _our_ bank

Reply:
In the API we say "Due to external providers' limitations, the reference will be truncated if length is greater than 16 characters."

Reply:
Thanks @hhurtadoWK that's probably better.